client/src/App.js

```diff
@@ -21,24 +25,47 @@ const theme = createTheme({
 });
 
 const App = () => {
+  const setUser = useSetRecoilState(userState);
   const currentUser = useRecoilValue(userState);
+  const [isLoadingUser, setIsLoadingUser] = useState(true);
+
+  const fetchUser = useCallback(async () => {
+    setIsLoadingUser(true);
+    console.log("fetchUser");
+    const access_token = localStorage.getItem("access_token");
+    const username = localStorage.getItem("username");
+    console.log(access_token, username);
+
+    if (!access_token) {
+      setUser(null);
+    } else {
+      const user = await getUserLogin();
+      console.log("test", user);
+
+      if (user?.data) {
+        localStorage.setItem("username", user.data.username);
+        setUser(user);
+      } else {
+        setUser(null);
+        localStorage.removeItem("access_token");
+        localStorage.removeItem("username");
+      }
+    }
+    setIsLoadingUser(false);
+  }, [setUser]);
+
+  useEffect(() => {
+    fetchUser();
+  }, [fetchUser]);
 
   return (
     <ThemeProvider theme={theme}>
       <Box>
         <Router>
           <AppHeader />
           <Routes>
-            <Route path="/" 
-              element={
-                !isEmpty(currentUser) ? (
-                  <Navigate to="/followed"/>
-                ) : (
-                  <Navigate to="/login" />
-                )
-              } 
-            />
-            <Route path="/login" element={<div>please login </div>} />
+            <Route path="/" element={<Home />} />
```

Comment:
Switched base path to a home component as we will implement a welcome page in the future and a base path is required for authentication